doc/src/tutorial/non_commutative.rst

```diff
@@ -0,0 +1,78 @@
+is_commutative == False:
+========================
+
+Passing commutative = False as argument in some operations, may lead to unexpected answers.
+For ex:
+	>>> x,y = symbols("x y")
+	>>> a,b = symbols("a b",commutative = True)
+	>>> c,d = symbols('c d',commutative = False)
+	>>> pprint((x**(a+b)).expand())
+	    a  b
+	   x ⋅x 
+	>>> pprint((x**(c+d)).expand())  
+	  c + d
+	 x
+	
+So x**(a + b) ! = (x**a*x**b) when a and b are non commutative.
+This is because here the expansion is not true in general, so SymPy does not perform it.
+
+Here's a proof which shows:
+	>>>pprint(exp(a + b).expand())
```

Comment:
Being your first Pr , i'll let you know how you can help yourself in such situations if you're stuck . Try to make sure that all tests pass locally before you ask for reviews . It's good use of both your time and the reviewers time , so that they can review properly . Otherwise they'll also have to help you with fixing failing test cases which might be due to really minor issues that you might have overlooked 

If you still can't figure out why the tests are failing, surely the community will help you out !

Currently the issue is that you need to add a blank here in line 20 after `>>>` . Actually not only in line 20 , you need to fix this in couple of other lines too like 23 etc .

Reply:
Thanks a lot! I'll fix that. 
But how do i know if something like a documentation passes all the tests? I mean how do i run the tests?

Reply:
Oops , I meant you have to add a blank space after each `>>>` not a blank line but that's okay . You should be going through the checks below and make changes accordingly , I'm not quite sure why you added a blank line .

Yeah ,you need to run tests locally before committing ( atleast those tests which involve the changes you've made for the Pr) . If all tests pass you can commit and if you encounter some exception or a test fails you'll find a DO NOT COMMIT message displayed on your screen .
For more info check this out https://github.com/sympy/sympy/wiki/Development-workflow and read how to run tests locally !